mowgli_etl/pipeline/wdc/wdc_generic_size.py

```diff
@@ -6,7 +6,20 @@
 
 @dataclass_json
 @dataclass
-class WdcGenericSize:
+class WdcProductSize:
+    """
+    :field name: Product type name pulled from WdcProductType.expected
+    :type name: str
+    :field dimension: Generic product dimensions
+    :type dimension: class<WdcProductDimensions>
```

Comment:
I guess this field is the accumulator, then you divide by the count?

Reply:
I was treating that as the average, but I could make it be an accumulator. The current implementation multiplies this value by the count, adds the new value, and divides by (count+1)

Reply:
Either way is fine for now. What matters is whether it produces intuitively accurate results.